packages/react-notifications/src/components/InAppMessaging/CloseIconButton/CloseIconButton.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
+
+import { Button } from '@aws-amplify/ui-react';
+import { IconClose } from '@aws-amplify/ui-react/internal';
```

Comment:
Is this okay?

Reply:
IMO this is an expected usage, but someone else (cc @reesscot or @zchenwei) can verify

Reply:
Yes, the icons are for internal use only. This means we can use them, but discourage them being used directly by customers.